enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -120,7 +118,21 @@ <h1>Linked learners</h1>
       {% endfor %}
       </tbody>
     </table>
-
+    {% if learners.has_other_pages %}
+      <p class="paginator">
+        {% for page_number in learners.paginator.page_range %}
+          {% if page_number == '.' %}
```

Comment:
This is weird. Comparing an integer variable with string?

Reply:
Adopted from https://github.com/django/django/blob/94a180402c9e52bca2e911b6427c88f436e95449/django/contrib/admin/templatetags/admin_list.py#L34